### ISSUE TITLE
Keep git and issue tracking history

### ISSUE THREAD

**Issue by Krinkle** [wikimedia/jquery.i18n#208]
From <https://github.com/wikimedia/jquery.i18n/issues/205>

I thought that the `santhoshtr/banana-i18n` repo was a temporary fork that would eventually be merged into here as a pull request. Note that the repos on GitHub can be easily renamed (and all URLs etc. will be maintained as redirects automatically by GitHub).

That way we would preserve history of commits, issues, pull requests, comments etc. and removes any ambiguity from things like "issue 123".

I see that banana-i18n is now no longer a fork but was re-created as repo under `wikimedia/` on Nov 2018, without history.

Would you be open to consolidating the forks? I'd be happy to help with that. The jQuery plugin could be maintained for a short period of time, either as-is, or as smaller wrapper around the standalone version. Is that feasible? Alternatively, it can be just as now a separate source directory, but at least the Git history would be preserved in some form then.

**Comment by santhoshtr** [comment]
Yes, I am open to explore that feasibility. jquery plugin being in a folder looks a good idea.

We need to keep the npm package publishing workflow(https://www.npmjs.com/package/banana-i18n and https://www.npmjs.com/package/@wikimedia/jquery.i18n)

**Comment by Krinkle** [comment]
I won't have time for this I'm afriad. I do think it would be a good idea still to have these exist in the same repo, and mainly concentrate on this repo (jquery.i18n) to preserve the history.

Once merging in the data and changes from banana-i18n you can rename the repos on GitHub (I can help with that) so that e.g. `banana-i18n` becomes `banana-i18n-archived` and `jquery.i18n` becomes `banana-i18n`. All jquery.i18n-related URLs about commits, issues, pulls etc would all continue to work and automatically redirect.
